setup.cfg

```
@@ -17,5 +17,5 @@
 [options.entry_points]
 cylc.configure = 
     configure = cylc.rose.rose:get_rose_vars
-cylc.rose_fileinstall =
+cylc.fileinstall =
```

This entry point can be used for more than file installation, for example https://github.com/cylc/cylc-flow/issues/3780 would require installing packages into virtual environments.

```suggestion
cylc.post_install =
```